Electron-Basic-Screen-Sharing/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "electron-basic-screen-sharing-v17",
```

Comment:
How do you feel about calling this just "electron-basic-screen-sharing" instead?  Saves us the hassle of updating the name should Electron break again, for example.

Reply:
Yep! I updated the branches' versions too.